var/spack/repos/builtin/packages/bazel/package.py

```diff
@@ -157,6 +157,9 @@ class Bazel(Package):
     patch('disabledepcheck.patch', when='@0.3.2:+nodepfail')
     patch('disabledepcheck_old.patch', when='@0.3.0:0.3.1+nodepfail')
 
+    # include-what-you-use violation that snuck under the radar until GCC 11
+    patch('ijar_limits_fix.patch', when='@0.3.2:5.0.0')
```

Comment:
Version ranges as inclusive. If you want >= 0.3.2, < 5, that would look like `0.3.2:4`